withings_api/common.py

```diff
@@ -840,6 +908,56 @@ def get_measure_value(
     )
 
 
+def new_heart_get_response(data: dict) -> HeartGetResponse:
+    """Create GetSleepResponse from json."""
+    return HeartGetResponse(
+        signal=tuple(sample for sample in data.get("signal", [])),
+        sampling_frequency=int_or_raise(data.get("sampling_frequency")),
+        wearposition=new_heart_wear_position(data.get("wearposition")),
+    )
+
+
+def new_heart_list_ecg(data: dict) -> HeartListECG:
+    """Create HeartListECG from json."""
+    return HeartListECG(
+        signalid=int_or_raise(data.get("signalid")),
+        afib=int_or_raise(data.get("afib")),
+    )
+
+
+def new_heart_blood_pressure(
+    data: Optional[Dict[Any, Any]]
+) -> Optional[HeartBloodPressure]:
+    """Create HeartBloodPressure from json."""
+    if data is None:
+        return data
+
+    return HeartBloodPressure(
+        diastole=int_or_raise(data.get("diastole")),
+        systole=int_or_raise(data.get("systole")),
+    )
+
+
+def new_heart_list_serie(data: dict) -> HeartListSerie:
+    """Create HeartListSerie from json."""
+    return HeartListSerie(
+        model=new_heart_model(int_or_raise(data.get("model"))),
+        ecg=new_heart_list_ecg(data.get("ecg", {})),
```

Comment:
Looks like this should be `dict_or_raise(data.get("ecg"))` since `new_heart_lists_ecg()` raises when it's values are not present.